app/models/physical_server_profile_template/operations.rb

```diff
@@ -0,0 +1,5 @@
+module PhysicalServerProfileTemplate::Operations
+  extend ActiveSupport::Concern
+
+  include_concern 'Assignment'
+end
```

Comment:
Do we need this at all if it is blank?  Same question with the `Assignment` module

Reply:
I deleted it.